internal/provider/newrelic/provider_integration_test.go

```diff
@@ -123,6 +126,49 @@ func Test_Getting_external_metric_generates_a_query_not_rejected_by_backend(t *t
 	})
 }
 
+//nolint:paralleltest // This test registers environment variables, so it must not be run in parallel.
+func Test_Getting_external_metric_through_proxy(t *testing.T) {
+	ctx := testutil.ContextWithDeadline(t)
+
+	t.Run("ends_with_error_when_proxy_fails", func(t *testing.T) {
+		port := 1337
```

Comment:
why using a local scoped variable for this? I think this can be moved out and be reused in the test below

Reply:
Agree, a leftover from debugging the problems with parallel tests where I tried to set different env variable hosts per subtest; fixed.